soap/src/java/com/zimbra/soap/mail/type/ActionSelector.java

```diff
@@ -187,7 +188,7 @@ public static ActionSelector createForIdsAndOperation(String ids, String operati
     public void setColor(Byte color) { this.color = color; }
     public void setName(String name) { this.name = name; }
     public void setFlags(String flags) { this.flags = flags; }
-    public void setNonExistentIds(String ids) { this.nonExistentIds = ids; };
+    public void setNonExistentIds(ZmBoolean r) { this.nonExistentIds = r; };
```

Comment:
Take a look at the documentation or look at other classes that use ZmBoolean.
The methods you create should abstract away the difference between booleans or Booleans and ZmBoolean